hack/make-rules/test-cmd-util.sh

```diff
@@ -2574,6 +2575,22 @@ run_rs_tests() {
   fi
 }
 
+run_daemonset_tests() {
```

Comment:
I see. The validation unit test only tests the validation part. I still want to test that after updating a DaemonSet with a semantically equal change, templateGeneration won't change. It can't be done in DaemonSet controller unit test, because templateGeneration is updated by API server but not DaemonSet controller.